tools/src/scie_pants/install_pants.py

```diff
@@ -78,22 +79,30 @@ def install_pants_from_pex(
     ptex: Ptex,
     extra_requirements: Iterable[str],
     find_links: str | None,
+    bootstrap_urls_path: str | None,
 ) -> None:
     """Installs Pants into the venv using the platform-specific pre-built PEX."""
     uname = os.uname()
-    major, minor = sys.version_info[:2]
-    pex_name = (
-        f"pants.{version}-cp{major}{minor}-{uname.sysname.lower()}_{uname.machine.lower()}.pex"
-    )
+    pex_name = f"pants.{version}-cp39-{uname.sysname.lower()}_{uname.machine.lower()}.pex"
```

Comment:
Drive-by fix where we hardcode `cp39` because At some point Pants will run on 3.11, but this PEX can run on anything. SO hardcoding is safer.